changes/2030.feature

```diff
@@ -0,0 +1,3 @@
+ClientRedirectError is no longer raised
```

Comment:
As we don't publish aiohttp release with `ClientRedirectError` the name should be changed to `RuntimeError` (used to be raised in aiohttp 2.2)